CC/Sounder/Radio.cc

```diff
@@ -114,11 +113,11 @@ int Radio::activateRecv(const long long rxTime, const size_t numSamps, int flags
         SOAPY_SDR_WAIT_TRIGGER | SOAPY_SDR_END_BURST
     };
     int flag_args = soapyFlags[flags];
-    // for USRP device start rx stream 1 sec in the future
+    // for USRP device start rx stream SETTLE_TIME_SEC sec in the future
```

Comment:
I think the name `SETTLE_TIME_SEC` is confusing in context. `SETTLE_TIME_MS` currently used in calibrate functions are literally the time used to wait for frequency and gains to be settled to the new value after changing them. If the purpose is to not use magic numbers, we could use a different name such as `NS_IN_SEC` or something of that sort here.

Reply:
that makes sense. but what do you mean by "NS" in "NS_IN_SEC"...?

Reply:
Oh I mean nanosec. It doesn't have to be this name. I guess we are looking for something more meaningful.

Reply:
fixed